client/src/Pages/LoginScreen.jsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import {Input, Button} from 'antd';
-import { Link } from 'react-router-dom';
+import {Row, Col} from 'antd';
+import LoginForm from './LoginForm';
 
 export default class LoginScreen extends React.Component{
 
     render(){
         return(
-            <div>
-                Please Enter Your Name: 
-                <Input placeholder = "Player Name"/>
-                <Link to="/player/waiting">
-                <Button>Login</Button>
-                </Link>
-            </div>
+            <>
+            <Row style = {{paddingTop: "3vh"}}>
+                <Col xs = {0} md = {8}/>
+                <Col xs = {24} md = {8} style = {{textAlign: "center", fontFamily: "minecraft", fontSize: "200%"}}>
+                    Please Enter Your Name: 
+                </Col>
+                <Col xs = {0} md = {8}/>
+            </Row>
+            <LoginForm/>
```

Comment:
Should this be put in its own row? I actually don't know, I just learned about antd's grid system for the first time while looking at this PR.

Reply:
I am going to leave with the Row convention, since the LoginForm renders in the same grid format to keep the spacing the same and uniform